tasks/extract_filepaths_for_granule/extract_filepaths_for_granule.py

```diff
@@ -137,10 +134,10 @@ def should_exclude_files_type(granule_url: str, exclude_file_types: List[str]) -
     for file_type in exclude_file_types:
         # Returns the first instance in the string that matches .ext or None if no match was found.
         if re.search(f"^.*{file_type}$", granule_url) is not None:
-            LOGGER.info(f"The file {granule_url} will not be restored because it matches the excluded file type {file_type}.")
+            LOGGER.debug(f"The file {granule_url} will not be restored because it matches the excluded file type {file_type}.")
```

Comment:
I would keep this as `info` or `warn` instead of `debug`. We want a user to know that something was not added to the restore list for a valid reason.